### PR TITLE
Remove all usage of read replicas enabled flag

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

**Review by alaister — APPROVED** [review]
LGTM!

File: apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityActions.tsx [hunk 1]
```diff
@@ -88,40 +82,32 @@ const UtilityActions = ({
             type="text"
             onClick={() => prettifyQuery()}
             className="px-1"
-            icon={<IconAlignLeft size="tiny" strokeWidth={2} className="text-gray-1100" />}
+            icon={<AlignLeft size="tiny" strokeWidth={2} className="text-gray-1100" />}
```
Comment (MildTomato) [inline apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityActions.tsx:85]: unsupported tailwind color

File: apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx [hunk 2]
```diff
@@ -62,136 +56,136 @@ export const UsePoolerCheckbox = ({
...
+      <div className="flex gap-x-3 items-center">
+        <Checkbox_Shadcn_
+          id={`use-pooler-${id}`}
+          checked={checked}
+          onCheckedChange={() => onCheckedChange(!checked)}
+        />
+        <div className="-mt-[2px] flex flex-col gap-y-1 w-full">
+          <div className="flex items-center gap-x-4">
+            <div className="flex items-center gap-x-2">
+              <label htmlFor={`use-pooler-${id}`} className="text-sm cursor-pointer">
```
Comment (MildTomato) [inline apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx:69]: `<Label_shadcn_>`

File: apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx [hunk 3]
```diff
@@ -62,136 +56,136 @@ export const UsePoolerCheckbox = ({
...
+                    {isSuccess && (
+                      <div className="flex items-center gap-x-1">
+                        <DropdownMenu open={open} onOpenChange={setOpen}>
+                          <DropdownMenuTrigger asChild>
+                            <Button
+                              type="outline"
+                              className="py-0.5 pr-1.5"
+                              iconRight={<ChevronDown strokeWidth={1} />}
+                            >
+                              <span className="capitalize">Mode: {poolingMode}</span>
```
Comment (MildTomato) [inline apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx:85]: why `<span>` ?

File: apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx [hunk 4]
```diff
@@ -62,136 +56,136 @@ export const UsePoolerCheckbox = ({
...
+                            </Button>
+                          </DropdownMenuTrigger>
+                          <DropdownMenuContent side="bottom" align="start" className="w-[280px]">
+                            <DropdownMenuItem
+                              key="session"
+                              className="flex items-center justify-between"
+                              onClick={() => onSelectOption('session')}
+                            >
+                              <span>Session mode</span>
```
Comment (MildTomato) [inline apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx:94]: dont need span

File: apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx [hunk 5]
```diff
@@ -62,136 +56,136 @@ export const UsePoolerCheckbox = ({
...
+                              {poolingMode === 'session' && (
+                                <Check size={16} className="text-brand" />
+                              )}
+                            </DropdownMenuItem>
+                            {poolingConfiguration?.pool_mode === 'transaction' && (
                               <DropdownMenuItem
```
Comment (MildTomato) [inline apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx:100]: should be using `<DropdownMenuCheckboxItem/>`

File: apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx [hunk 6]
```diff
@@ -62,136 +56,136 @@ export const UsePoolerCheckbox = ({
...
+                            <Button
+                              type="outline"
+                              className="py-0.5 pr-1.5"
+                              iconRight={<ChevronDown strokeWidth={1} />}
+                            >
+                              <span className="capitalize">Mode: {poolingMode}</span>
+                            </Button>
+                          </DropdownMenuTrigger>
+                          <DropdownMenuContent side="bottom" align="start" className="w-[280px]">
+                            <DropdownMenuItem
```
Comment (MildTomato) [inline apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx:89]: should be using `<DropdownMenuCheckboxItem/>`

File: apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx [hunk 7]
```diff
@@ -62,136 +56,136 @@ export const UsePoolerCheckbox = ({
...
-                      : resolvesToIpV6
-                        ? 'Resolves to IPv6'
-                        : 'Will resolve to IPv6'}
-                  </Badge>
-                )}
-                {pgBouncerStatus?.active && (
-                  <Badge variant="warning">PgBouncer pending removal</Badge>
-                )}
-              </div>
+                            <Separator className="my-1" />
```
Comment (MildTomato) [inline apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx:153]: `<DropdownMenuSeparator/>` please

**Review by MildTomato — CHANGES_REQUESTED** [review]

File: apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityActions.tsx (hunk 1 above)
Comment (MildTomato) [reply apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityActions.tsx:85]: probably text-foreground-muted, but now sure what it looks like

File: apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx (hunk 7 above)
Comment (MildTomato) [reply apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx:153]: what's with the `my-1` on seperators ?

File: apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityActions.tsx (hunk 1 above)
Comment (joshenlim) [reply apps/studio/components/interfaces/SQLEditor/UtilityPanel/UtilityActions.tsx]: i dont think we need that class name actually - will remove

**Review by MildTomato — COMMENTED** [review]
think this might not work

File: apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx [hunk 8]
```diff
@@ -62,136 +62,129 @@ export const UsePoolerCheckbox = ({
...
+                              iconRight={<ChevronDown strokeWidth={1} />}
+                            >
+                              Mode: {poolingMode}
+                            </Button>
+                          </DropdownMenuTrigger>
+                          <DropdownMenuContent side="bottom" align="start" className="w-[280px]">
+                            <DropdownMenuRadioGroup
+                              value={poolingMode}
+                              onValueChange={(value) =>
+                                onSelectOption(value as 'session' | 'transaction')
```
Comment (MildTomato) [inline apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx:98]: we not have actual type variable?

Comment (MildTomato) [reply apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx:98]: non blocking

**Review by MildTomato — APPROVED** [review]

File: apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx (hunk 8 above)
Comment (joshenlim) [reply apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx:98]: actually good point haha i should just type the value instead

but the type of value from onValueChange is just string
image

**Pull request merged**